sms/send-signed-sms.py

```diff
@@ -0,0 +1,33 @@
+#Import dependencies
+import nexmo, os
+from os.path import join, dirname
+from dotenv import load_dotenv
+
+#Load the environment
+envpath = join(dirname(__file__),'./.env')
+load_dotenv(envpath)
+
+#Init the client
+client = nexmo.Client(
+    key = os.getenv('NEXMO_API_KEY'),
+    secret = os.getenv('NEXMO_API_SECRET'),
```

Comment:
They shouldn't need to add an API Secret if they're signing (the signature acts as the auth)

Reply:
Took after the other code snippet models. My bad. Cleaning up